Gruntfile.js

```diff
@@ -38,6 +38,12 @@ module.exports = function(grunt) {
     'mochaTest:functional',
   ]);
 
+  grunt.registerTask('test-appveyor', [
```

Comment:
I would rather not introduce another execution path through the test suite unless we have to. It creates a maintenance burden. 

Can we just create a wrapper around the flowbin task that skips the test? It could just look for an appveyor env var and skip [like this one](https://github.com/rpl/web-ext/blob/3c7a2c7cae90a589b7d9ad6dbc9fe1f82ef60c84/Gruntfile.js#L53).

I don't mind running eslint on Windows because that's what devs will be doing locally. It would be good to have coverage that it works on Windows.